### DISCUSSION TITLE
Inserting multiple images inline but centered

### DISCUSSION THREAD

**Discussion by TobiasBaeumlin** [McShelby/hugo-theme-relearn#811] [Q&A] [answered]
I wonder if there's a simple way to insert several images in one row like with 'classes=inline' but centered. 
And / or increase the margin between inline images.

TIA
Tobias

**Comment by McShelby** [comment, marked as answer]
No, not yet. Inline causes all images to be inserted into text flow which causes them to be aligned left (or right if you are using RTL text direction)
